client/plots/violin.interactivity.js

```diff
@@ -89,23 +86,18 @@ export function setInteractivity(self) {
 				callback: async () => self.listSamples(event, t1, t2, plot, start, end)
 			})
 		}
-		plot.start = start
-		plot.end = end
-		self.displayMenu(event, options, plot)
+		self.displayMenu(event, options, plot, true, start, end)
 		// const brushValues = plot.values.filter(i => i > start && i < end)
 	}
 
-	self.displayMenu = function (event, options, plot) {
+	self.displayMenu = function (event, options, plot, isBrush, start, end) {
```

Comment:
can delete `isBrush`. if start and stop are provided, show the brush range

Reply:
okay. done